core/src/main/java/org/bitcoinj/core/NetworkParameters.java

```diff
@@ -661,7 +661,7 @@ public EnumSet<Script.VerifyFlag> getTransactionVerificationFlags(final Block bl
     public abstract int getProtocolVersionNum(final ProtocolVersion version);
 
     public static enum ProtocolVersion {
-        MINIMUM(70220),
+        MINIMUM(70227),
```

Comment:
This will prevent connecting to older and non-upgrade Dash Core 18 or lower nodes.

This will avoid following the wrong fork in the chain after Core 19 features are activated with a hard fork.